includes/class-wc-payments-utils.php

```diff
@@ -716,59 +716,163 @@ public static function get_filtered_error_status_code( Exception $e ): int {
 	}
 
 	/**
-	 * Retrieves Stripe minimum order value authorized per currency.
-	 * The values are based on Stripe's recommendations.
-	 * See https://docs.stripe.com/currencies#minimum-and-maximum-charge-amounts.
+	 * Get the BNPL limits per currency for a specific payment method.
 	 *
-	 * @param string $currency The currency.
-	 *
-	 * @return int The minimum amount.
+	 * @param string $payment_method The payment method name ('affirm', 'afterpay_clearpay', or 'klarna').
+	 * @return array The BNPL limits per currency for the specified payment method.
 	 */
-	public static function get_stripe_minimum_amount( $currency ) {
-		switch ( $currency ) {
-			case 'AED':
-			case 'MYR':
-			case 'PLN':
-			case 'RON':
-				$minimum_amount = 200;
-				break;
-			case 'BGN':
-				$minimum_amount = 100;
-				break;
-			case 'CZK':
-				$minimum_amount = 1500;
-				break;
-			case 'DKK':
-				$minimum_amount = 250;
-				break;
-			case 'GBP':
-				$minimum_amount = 30;
-				break;
-			case 'HKD':
-				$minimum_amount = 400;
-				break;
-			case 'HUF':
-				$minimum_amount = 17500;
-				break;
-			case 'JPY':
-				$minimum_amount = 5000;
-				break;
-			case 'MXN':
-			case 'THB':
-				$minimum_amount = 1000;
-				break;
-			case 'NOK':
-			case 'SEK':
-				$minimum_amount = 300;
-				break;
```

Comment:
Mentioned in the PR description, this was [added to handle BNPL minimum amounts](https://github.com/Automattic/woocommerce-payments/pull/9355) but is actually Stripe minimum amounts. It's was only used for that purpose and is safe to remove.